docs/geedocs/5.3.5/answer/releaseNotes/relNotesGEE5_3_5.html

```diff
@@ -73,9 +73,6 @@ <h1>Release notes: Open GEE 5.3.5<a class="headerlink" href="#release-notes-open
 <div class="docutils container">
 <div class="content docutils container">
 <p>Open GEE 5.3.5 is currently in development</p>
-<p class="rubric">New Features</p>
-<p><strong>New feature here</strong>.
-New feature description goes here.</p>
 <p class="rubric">Supported Platforms</p>
 <p>The Open GEE 5.3.5 release is supported on 64-bit versions of the
```

Comment:
we can remove "Open GEE 5.3.5 is currently in development"

Reply:
done